samples/MvvmSample.Core/ViewModels/MessengerPageViewModel.cs

```diff
@@ -27,7 +45,7 @@ public string Username
 
             protected override void OnActivated()
             {
-                Messenger.Register<UserSenderViewModel, CurrentUsernameRequestMessage>(this, (r, m) => m.Reply(r.Username));
+                Messenger.Register<CurrentUsernameRequestMessage>(this, (_, m) => m.Reply(Username));
```

Comment:
Why has this been changed?
This new version is incorrect and will cause a closure to be created in the handler.

Reply:
Hmm, I copied these ViewModels over from the work that @nickrandolph did for the Uno samples. I'll look into it more and see what I did here

Reply:
Yeah that was incorrect as well (I think I had left a review there already). Was probably either just an oversight or just code that had been written before the last few updates to that interface, where I added this new signature 🙂